run.py

```diff
@@ -117,6 +118,10 @@
         from bugbug.models.stepstoreproduce import StepsToReproduceModel
 
         model_class = StepsToReproduceModel
+    elif args.goal == "regressionrange":
+        from bugbug.models.regressionrange import RegressionRangeModel
+
+        model_class = RegressionRangeModel
```

Comment:
This code changed a bit, you'll need to rebase on top of master and modify bugbug/models/__init__.py instead.